blakserv/webhook.h

```diff
@@ -0,0 +1,72 @@
+// Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+// All rights reserved.
+//
+// This software is distributed under a license that is described in
+// the LICENSE file that accompanies it.
+//
+// Meridian is a registered trademark.
+/*
+* webhook.h
+*
+
+  Cross-platform webhook message delivery system for Meridian 59 server.
+
+  ARCHITECTURE:
+  This system enables the M59 server to send real-time notifications (webhooks) to external
+  services like Discord without blocking game performance. It uses a client-server pipe model:
+  
+  [M59 Server] --pipe--> [Webhook Listener] --HTTP--> [Discord/Webhooks]
+
+  MULTI-SERVER SUPPORT:
+  - External webhook listeners create multiple pipe servers (webhookN, where N=1-10)
+  - Each M59 server instance automatically claims an available pipe
+  - Up to 10 concurrent servers supported per webhook listener instance
+  - No configuration needed - automatic pipe distribution
+
+  PERFORMANCE:
+  - Pipes are opened once at startup and kept open (not per-message)
+  - Non-blocking operations prevent game server blocking
+  - Only 1 system call per message vs 3 in naive implementation
```

Comment:
I don't understand this last comment--you're documenting an implementation that doesn't exist?